.github/workflows/daily.yml

```diff
@@ -561,7 +561,7 @@ jobs:
       run: |
         ./runtest-cluster --tls-module ${{github.event.inputs.cluster_test_args}}
 
-  test-centos7-tls-no-tls:
+  test-centos7-tls-module-no-tls:
```

Comment:
Could you give some reasons why you suggest test-centos7-tls rename to test-centos7-tls-module, and test-centos7-tls-no-tls rename as well, it confuses me, thanks

Reply:
because we use `make BUILD_TLS=module` (see below the make part) in these cases, so rename it is better i guess

Reply:
I see, Thanks